metric/bigtable/src/main/java/com/spotify/heroic/metric/bigtable/api/BigtableDataClientImpl.java

```diff
@@ -66,12 +66,12 @@ public BigtableDataClientImpl(
     }
 
     @Override
-    public AsyncFuture<List<Row>> readRows(
+    public AsyncFuture<List<FlatRow>> readRows(
         final String tableName, final ReadRowsRequest request
     ) {
-        return convertRows(session
+        return convert(session
                 .getDataClient()
```

Comment:
Not formatted correctly. This was wrongly formatted already in the old code. Lets fix it.